cakefuzzer/domain/vulnerability.py

```diff
@@ -15,6 +15,7 @@ def __hash__(self) -> int:
 
     timestamp: float  # When?
     detection_result: str  # What?
+    detection_location: str  # Where?
```

Comment:
I don't believe it makes sense to make this field mandatory for all types of scanners. 
Atm it only makes sense if we have html output so only in one of all available scanners.
Ergo this should probably we Optional[str]